src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/route/DataLoadRoute.java

```diff
@@ -49,15 +48,9 @@
     @Autowired
     CamelContext camelContext;
 
-    @Autowired
-    ArchiveAzureFileProcessor azureFileProcessor;
-
-
     @Autowired
     HeaderValidationProcessor headerValidationProcessor;
 
-
-    @SuppressWarnings("unchecked")
```

Comment:
have u checked other classes for SuppressWarnings annotation?
it should be added only if it is utmost required and not as a general practice